src/js/controls/Header.js

```diff
@@ -187,7 +450,14 @@ lm.utils.copy( lm.controls.Header.prototype, {
 		for( var i = 0; i < this.tabs.length; i++ ) {
 			this.tabs[ i ]._$destroy();
 		}
-    	$( document ).off('mouseup', this.hideAdditionalTabsDropdown);
+
+		this._detachWheelListener();
+		this._handleItemPickedDropped();
+
```

Comment:
Cancel animation frame here as well

Reply:
It's already cancelled inside the itemdropped function.